src/components/App/styles.scss

```diff
@@ -0,0 +1,3 @@
+.container, .main {
```

Comment:
`.wrapper, .container` as per above comments (haven't done as a commit suggestion here in-case there were problems)